ui/src/app/app-routing.tsx

```diff
@@ -74,11 +85,21 @@ export const AppRoutingComponent: React.FunctionComponent<{signIn: Function}> =
 })
 export class AppRouting extends ReactWrapperBase {
   constructor(private signInService: SignInService) {
-    super(AppRoutingComponent, ['signIn']);
+    super(AppRoutingComponent, ['onSignIn', 'signIn']);
+    this.onSignIn = this.onSignIn.bind(this);
     this.signIn = this.signIn.bind(this);
   }
 
+  onSignIn(): void {
+    this.signInService.isSignedIn$.subscribe((signedIn) => {
+      if (signedIn) {
+        return <Redirect to='/'/>;
+      }
+    });
+  }
+
   signIn(): void {
+    AnalyticsTracker.Registration.SignIn();
```

Comment:
This may be new since I wrote this Angular wrapping of the react router, but it was missing from here and whereas it was in the SignInComponent Angular wrapper